app/src/main/java/com/kylecorry/trail_sense/astronomy/infrastructure/commands/LunarEclipseAlertCommand.kt

```diff
@@ -42,21 +41,32 @@ class LunarEclipseAlertCommand(private val context: Context) : Command<Coordinat
             return
         }
 
-        val notification = Notify.status(
+        val notification = createNotification(eclipse)
+
+        Notify.send(context, NOTIFICATION_ID, notification)
+    }
+
+    private fun createNotification(eclipse: Eclipse): Notify.StatusNotification {
+        val notificationTitle = context.getString(R.string.lunar_eclipse)
```

Comment:
Can you inline these with named parameters? You can keep the separate method / constant for the notification ID, but I prefer how it looks with the parameters rather than local variables